tests/models/test_models_vae.py

```diff
@@ -214,21 +283,7 @@ def output_shape(self):
         return (3, 32, 32)
 
     def prepare_init_args_and_inputs_for_common(self):
-        init_dict = {
-            "in_channels": 3,
-            "out_channels": 3,
-            "down_block_types": ["DownEncoderBlock2D", "DownEncoderBlock2D"],
-            "down_block_out_channels": [32, 64],
```

Comment:
Possible to reduce the block out channel size? e.g. [4, 8]?

Reply:
Could do in a follow-up PR, but this would explode this PR and be a bit out of scope.